packages/commerce-sdk-react/src/auth/storage/cookie.ts

```diff
@@ -35,6 +35,18 @@ export class CookieStorage extends BaseStorage {
     }
     delete(key: string) {
         const suffixedKey = this.getSuffixedKey(key)
-        Cookies.remove(suffixedKey)
+
+        // We usually use `Cookies.remove(key)` from the `js-cookie` library to delete a cookie. But, this
+        // method fails when removing a cookie on an iframed website like within StorefrontPreview on WebKit
+        // browsers such as Chrome and Safari.
+
+        // Set the expiration date in the past to delete the cookie
+        document.cookie = `${suffixedKey}=; expires=Thu, 01 Jan 1970 00:00:01 GMT; secure; samesite=none`
```

Comment:
```suggestion
        Cookies.remove(suffixedKey, {...getDefaultCookieAttributes()})
```
All `Cookies.remove` does under the hood is set `document.cookie`, same as you did. So we can still use `Cookies.remove`, we just need to provide it with the same parameters that we used when calling `Cookies.set`. To that end, we might also want to add an optional `options` parameter, same as we have for `set`.
```suggestion
        Cookies.remove(suffixedKey, {
          ...getDefaultCookieAttributes(),
          ...options
        })
```

Reply:
Good catch  @wjhsf! It's better to use the same method to set and remove cookies with the same default attributes